src/Microsoft.DotNet.Darc/src/Darc/Options/UpdateSubscriptionCommandLineOptions.cs

```diff
@@ -14,6 +13,12 @@ class UpdateSubscriptionCommandLineOptions : CommandLineOptions
         [Option("id", Required = true, HelpText = "Subscription's id.")]
         public string Id { get; set; }
 
+        [Option("trigger", SetName = "triggering", HelpText = "Automatically trigger the subscription on update.")]
```

Comment:
Doesn't an default option makes sense here and so we'll only need one of these options?